docs/tutorial/troubleshooting/migrate_pool_by_migrating_disks.md

```diff
@@ -0,0 +1,314 @@
+|----------------------------------------------------------------------------------------------------------------------|
+```
+### Replaced Nodes
+  Performed node replacement action as specified [here](#reproduce-the-node-replacement-situation). Now the existing nodes were replaced with new nodes but the same disks were attached to the nodes(able to confirm from below outputs). Below are updated node details
+```sh
+$ kubectl get nodes --show-labels
+NAME                                           STATUS   ROLES    AGE     VERSION  LABELS
+ip-192-168-25-235.us-east-2.compute.internal   Ready    <none>   6m28s   v1.15.9  kubernetes.io/hostname=ip-192-168-25-235
+ip-192-168-33-15.us-east-2.compute.internal    Ready    <none>   8m3s    v1.15.9  kubernetes.io/hostname=ip-192-168-33-15
+ip-192-168-75-156.us-east-2.compute.internal   Ready    <none>   4m30s   v1.15.9  kubernetes.io/hostname=ip-192-168-75-156
+```
+
+Blockdevice names were not changed but node name and host names are updated with new node and hostname details
+```sh
+$ kubectl get bd -n openebs
+NAME                                           NODENAME                                       SIZE          CLAIMSTATE   STATUS   AGE
+blockdevice-4505d9d5f045b05995a5654b5493f8e0   ip-192-168-33-15.us-east-2.compute.internal    10737418240   Claimed      Active   3h8m
+blockdevice-798dbaf214f355ada15d097d87da248c   ip-192-168-25-235.us-east-2.compute.internal   10737418240   Claimed      Active   3h8m
+blockdevice-c783e51a80bc51065402e5473c52d185   ip-192-168-75-156.us-east-2.compute.internal   10737418240   Claimed      Active   3h8m
+```
+Once the nodes where replaced, the pool manager pods will remain in pending state due to existence of nodeselector on pool manager deployments. We can verify pool details using following command
+```sh
+$ kubectl get cspc -n openebs
+NAME         HEALTHYINSTANCES   PROVISIONEDINSTANCES   DESIREDINSTANCES   AGE
+cstor-cspc   0                  3                      3                  170m
+```
+For verifying pool manager pods
+```
+$ kubectl get po -n openebs -l app=cstor-pool
+NAME                               READY   STATUS    RESTARTS   AGE
+cstor-cspc-chjg-85f65ff79d-pq9d2   0/3     Pending   0          16m
+cstor-cspc-h99x-57888d4b5-kh42k    0/3     Pending   0          15m
+cstor-cspc-xs4b-85dbbbb59b-wvhmr   0/3     Pending   0          18m
+```
+In the above output **HEALTHYINSTANCES** were **0** and all the pool pods were in pending state because nodes were replaced with new nodes and still CSPC spec and cStor pool manager is pointing to old nodes.
+
+**NOTE**: Please make sure that PROVISIONEDINSTANCES and DesiredInstances count should match before performing steps.
+
+## Steps to bring cStor pool back online
+
+### Step1: Update validatingwebhookconfiguration resource failurePolicy
+
+This step is required to inform the kube-APIServer to ignore the error if kube-APIServer is not able to reach cStor admission server.
```

Comment:
Is this the only reason? Will the admission server allow node selector changes on CSPC?

Reply:
I will update

Reply:
Yes, If the admission server is available and if it rejects the request then irrespective of failurePolicy it will throw an error in front of users.

Reply:
For more info: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy